public/index.html

```diff
@@ -65,10 +69,12 @@ <h1>Notifications</h1>
             socket.emit('connect_user', `${user}`)
             socket.on('notification', (notification) =>  {
             const requestId = notification.requestID
-            const link = `${url}api/v1/trips/${requestId}`
-            $("#notifications").append(`<li><a href=${link} target="_blank">${notification.body}</li>`);
-            console.log(link);
-            
+            const link = `${url}api/v1/trips/${requestId}`;
+            if(notification.type == 'comment'){
+                const { names, body, createdAt } = notification
+                $("#notifications").append(`<li><a>${names} commented on your request: <span class="content">${body}</span></li>`);
```

Comment:
@RusimbiP Good job, but I had some concerns after testing your feature on review apps. I found that the notification link is not clickable!

Would you refactor it in the way you did it last time so that when I click on the notification link I'll be able to see the details of a specific comment?
I know we'll do much on the front end but I am saying this because you implemented it last time on ```Show notification for edited trip requests``` and it was fantastic cool. Look at the screen shot below, ```I am on review apps``` is not clickable!

<img width="557" alt="Screen Shot 2020-03-18 at 17 44 08" src="https://user-images.githubusercontent.com/51243858/76980382-c8bf6a80-6941-11ea-917d-a8673ee70b1e.png">

Reply:
I removed the link. Let's do it in the front end

Reply:
@RusimbiP That's interesting